### PR TITLE
Added review lifecycle info

### PR DESCRIPTION
**Pull request by torgo**
References https://github.com/w3ctag/process/issues/22

### CODE REVIEW

File: workmode/index.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ We also suggest you read the [Ethical Web Principles](https://www.w3.org/2001/ta
 
 A positive review doesn't mean that a spec is "supported" by the TAG or that the design is free of issues, but hopefully it'll help improve the designs of APIs that request feedback.
 
+The TAG will periodically triage new issues and make assignments. Typically two or more TAG members will be assigned to each issue.  Issue assignment means these are the TAG members reponsible for marshalling the activity on this issue.   We will also assign labels according to the category of review, its provenance, etc... Milestones are used to denote what week we will next discuss the issue.  We will close the issue when a review is complete.  We will also typically mark the review as *satisfied*, *satisified with concerns*, *unsatisified*. We may also used resolution codes *timed out* (if we did not hear back from the requestor in a timely fashion), *too early* if we judged it was too early to start a review, *validated* (meaning we didn't have any response), or *withdrawn* (denoting that the review was withdrawn by the requestor).
```
Comment (LeaVerou) [inline workmode/index.md:26]: - Way too long of a line, needs broken down into semantic chunks
- We used "timed out" today for a review that *we* didn't do in time

Comment (LeaVerou) [reply workmode/index.md:26]: In a discussion with @plinss just now we were wondering if we should also have a "too late" label, and some text that basically says "if a feature has shipped in every major browser, it's too late for TAG review, the ship has sailed". We should not be spending our extremely limited cycles on reviews that don't matter.

Comment (LeaVerou) [reply workmode/index.md:26]: Actually, I'll file an issue

Comment (torgo) [inline workmode/index.md:26]:
```suggestion
The TAG will periodically triage new issues and make assignments. Typically two or more TAG members will be assigned to each issue.  Issue assignment means these are the TAG members reponsible for marshalling the activity on this issue.   We will also assign labels according to the category of review, its provenance, etc... 

Milestones are used to denote what week we will next discuss the issue.  We will close the issue when a review is complete.  We will also typically mark the review as *satisfied*, *satisified with concerns*, *unsatisified*. We may also used resolution codes *timed out* (if we did not hear back from the requestor in a timely fashion), *too early* if we judged it was too early to start a review, *validated* (meaning we didn't have any response), or *withdrawn* (denoting that the review was withdrawn by the requestor).
```

File: workmode/index.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ We also suggest you read the [Ethical Web Principles](https://www.w3.org/2001/ta
 
 A positive review doesn't mean that a spec is "supported" by the TAG or that the design is free of issues, but hopefully it'll help improve the designs of APIs that request feedback.
 
+The TAG will periodically triage new issues and make assignments. Typically two or more TAG members will be assigned to each issue.  Issue assignment means these are the TAG members reponsible for marshalling the activity on this issue.   We will also assign labels according to the category of review, its provenance, etc... 
+
+Milestones are used to denote what week we will next discuss the issue.  We will close the issue when a review is complete.  We will also typically mark the review as *satisfied*, *satisified with concerns*, *unsatisified*. We may also used resolution codes *timed out* (if we did not hear back from the requestor in a timely fashion), *too early* if we judged it was too early to start a review, *validated* (meaning we didn't have any response), or *withdrawn* (denoting that the review was withdrawn by the requestor).
```
Comment (torgo) [inline workmode/index.md:28]:
```suggestion
Milestones are used to denote what week we will next discuss the issue.  We will close the issue when a review is complete.  We will also typically mark the review as *satisfied*, *satisified with concerns*, *unsatisified*. We may also used resolution codes *timed out* (if we did not hear back from the requestor in a timely fashion or if **we** did not reply in a timely enough fashion), *too early* if we judged it was too early to start a review, *validated* (meaning we didn't have any response), or *withdrawn* (denoting that the review was withdrawn by the requestor).
```

**Comment by torgo** [comment]
Hi Lea - I've made some changes to address the issues you mentioned and so will go ahead and merge this. We can address the "too late" issue separately.

**Pull request merged**